src/DocScan/Session/Retrieve/ReportResponse.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\DocScan\Session\Retrieve;
+
+class ReportResponse
+{
+
+    /**
+     * @var RecommendationResponse|null
+     */
+    private $recommendation;
+
+    /**
+     * @var BreakdownResponse[]
+     */
+    private $breakdown = [];
+
+    /**
+     * ReportResponse constructor.
+     * @param array<string, mixed> $reportData
+     */
+    public function __construct(array $reportData)
+    {
+        if (isset($reportData['recommendation'])) {
+            $this->recommendation = new RecommendationResponse($reportData['recommendation']);
+        }
+
+        if (isset($reportData['breakdown'])) {
+            $this->breakdown = $this->parseBreakdowns($reportData['breakdown']);
```

Comment:
Could replace private method with:
```suggestion
            foreach ($reportData['breakdown'] as $breakdown) {
                $this->breakdown[] = new BreakdownResponse($breakdown);
            }
```

Reply:
Fixed in 93f3254